R/compile_report.R

```diff
@@ -44,54 +72,17 @@
 
 compile_report <- function(file, quiet = FALSE, factory = getwd(),
                            clean_report_sources = FALSE,
-                           remove_cache = TRUE, 
+                           remove_cache = TRUE,
                            encoding = "UTF-8",
                            params = list(), ...) {
-
-  ## This function will:
-  
-  ## 1. perform cleaning of `report_sources/` if `clean_report_sources = TRUE`;
-  ## this will remove pretty much anything that's not an `Rmd` file or a
-  ## `README`; this is externalised in the function `clean_report_sources`
-
-  ## 2. check that input file exists; identify all files in `report_sources/`
-  ## prior to the compilation, used later to diff post-compilation to identify
-  ## new files
-  
-  ## 3. compile the user-provided report `foo_date[].Rmd` file stored in
-  ## `report_sources/`; compilation parameters are passed to
-  ## `rmarkdown::render`'s argument `params` through `render_param`; parameters
-  ## will also be used to form the output file name, which will display the
-  ## parameter names and first values, so the output folder will look like:
-  ## `compiled_foo_[foo-values]_bar_[bar-values]_[date]_[time]/`
-
-  ## 4. identify all files in report_sources; new ones are those which where not
-  ## around at step 1; this can cause problems when there are left-over outputs
-  ## from previous manual compilations inside `report_sources/`; it will be
-  ## often safest to use `clean_report_sources = TRUE` so that these left-overs
-  ## are removed before step 3
-
-  ## 5. move all outputs identified in step 3 to a dedicated folder in
-  ## report_outputs/; this folder will be named after the input report,
-  ## e.g. `foo_date/compiled_[date]_[time]/`, and will include `params` if
-  ## provided (see point 3).
-  
-  
-  ## These are sed to create log entry, passed in with the env (not called directly in the code)
```

Comment:
This through #86 are actually important to keep for logging purposes - the info is passed along as part of the environment to the log function (i'll add tests to make sure this doesn't disappear, and i'll edit line 80 to be more clear